sections/03_categorize.md

```diff
@@ -28,7 +28,148 @@ we share these data while respecting privacy? I've made a section for each of
 these. Feel free to add more. I see each section as something on the order of
 1-2 paragraphs in our context.*
 
-#### Standardization/integration
+#### Clinical Care
+
+#### Imaging Applications in Health Care
+
+One of the general areas where deep learning methods have had substantial
+success has been in image analysis. Applications in areas of medicine that use
+imaging extensively are also emerging. Mammography has been one area with
+numerous contributions [@doi:10.1007/978-3-319-46723-8_13,
+@doi:10.1007/978-3-319-24553-9_74, @doi:10.1101/095794, @doi:10.1101/095786]. In
+all of this work, the researchers must work around a specific challenge - the
+limited number of well annotated training images. To expand the number and
+diversity of images, the researchers have employed approaches where they employ
+adversarial examples [@doi:10.1101/095786] or first train towards human-created
```

Comment:
Got it.  You're right, my mind jumped to Generative Adversarial Networks when I saw adversarial.